taipy/gui/server.py

```diff
@@ -337,12 +337,14 @@ def run(
             log = logging.getLogger("werkzeug")
             log.disabled = True
             if not is_running_from_reloader():
```

Comment:
Why the test?

Reply:
different messages as you can see. The test is for the reloader during dev mode